tests/integ/scala/test_function_suite.py

```diff
@@ -2823,6 +2824,34 @@ def test_array_append(session):
     )
 
 
+@pytest.mark.skipif(
+    "config.getoption('local_testing_mode', default=False)",
+    reason="array_remove is not yet supported in local testing mode.",
+)
```

Comment:
@sfc-gh-aling / @sfc-gh-jrose do we have a jira to support array functions? I believe there are multiple array related functions not supported in local testing mode.

Reply:
We do not currently have a jira for this. I can add one to our backlog.